apps/codeforafrica/src/pages/api/admin/config.js

```diff
@@ -1,5 +1,117 @@
 const APP_DIRECTORY = process.env.NEXT_PUBLIC_APP_DIRECTORY;
 
+const seoObject = {
+  label: "SEO",
+  name: "seo",
+  widget: "object",
+  fields: [
+    {
```

Comment:
Lets include title template here as well to support child pages such as /about/partners that need to update the template.

```suggestion
    {
      label: "Title Template",
      name: "title-template",
      widget: "string",
      hint: 'Uses title template from Settings | General if not provided',
    },
    {
```